src/Sylius/Component/User/Model/AdvancedUserInterface.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Component\User\Model;
+
+use Symfony\Component\Security\Core\User\UserInterface as SymfonyUserInterface;
+
```

Comment:
Missing uses for docblock exceptions. 

I didn't check if those exception still exists.

```php
use Symfony\Component\Security\Core\Exception\AccountExpiredException;
use Symfony\Component\Security\Core\Exception\AccountStatusException;
use Symfony\Component\Security\Core\Exception\CredentialsExpiredException;
use Symfony\Component\Security\Core\Exception\DisabledException;
use Symfony\Component\Security\Core\Exception\LockedException;
```

Reply:
fixed